src/test/java/service/CheckoutControllerTest.java

```diff
@@ -39,7 +39,16 @@
 
     @BeforeClass
     public static void setupConfig() {
-        Application.gateway = BraintreeGatewayFactory.fromConfigFile(new File("config.properties"));
+        File configFile = new File("config.properties");
```

Comment:
Yeah, that would make sense. It's a little tricky because it's not entirely duplicated (we don't want to System.exit in the tests), but that seems valuable to refactor. That's a bit out of scope for this PR, so we will follow up with another. :+1: